src/BallotParser.php

```diff
@@ -37,30 +34,6 @@ public function parse(string $text) : Ballot
         return $ballot;
```

Comment:
Can we combine this with the minor changes I checked in to 0.x?  I had started using NBallots so that I could use the results for RankedPairsCalculator.  Otherwise, this cleanup looks great!

Reply:
Yep!